web/src/routes.rs

```diff
@@ -13,5 +14,9 @@ pub fn routes(app_state: AppState) -> axum::Router {
         .route("/login", get(users::login::get).post(users::login::post))
         .route("/signup", get(users::signup::get).post(users::signup::post))
         .route("/logout", get(users::login::logout))
+        .route(
+            "/insert_article",
+            post(users::add_url::insert_article_handler),
+        )
         .with_state(app_state)
```

Comment:
we added route called "/insert_article" and it sends a POST request to the "/insert_article" endpoint.